angular/src/app/make-post/make-post.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { FormGroup } from '@angular/forms';
+import { MatChipInputEvent } from '@angular/material/chips';
+import { COMMA, ENTER } from '@angular/cdk/keycodes';
+
+export interface Category{
```

Comment:
should this be extracted to the Post interface?

Reply:
to clarify, I meant the category interface. although I'm not sure if this was just used with the chip thing